deployer/Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN microdnf update -y && \
     && microdnf clean all && \
     rm -rf /var/cache/yum
 
+RUN mkdir -p sc-override
```

Comment:
Do we really need this, doesn't the config map to mount mapping take care of it?

Reply:
Thanks. Yes, the config map takes care of it.